webapp/src/main/java/io/github/microcks/util/ai/AICopilotHelper.java

```diff
@@ -388,4 +388,100 @@ private static JsonNode followRefIfAny(JsonNode spec, JsonNode referencableNode)
    private static JsonNode getNodeForRef(JsonNode spec, String reference) {
       return spec.at(reference.substring(1));
    }
+
+  /**
+   * Some AI will reuse already present examples and x-microcks-operation in the
+   * spec. This method cleans a specification from its examples.
+   */
+  protected static String removeTagsFromOpenAPISpec(String specification, String operationName) throws Exception {
```

Comment:
I thought at first that this method was dealing with OpenAPI `tags` within a spec. Maybe `removeTokensFromOpenAPISpec()` would be a better name... Also, as I understood it, it will replace the `removeExamplesFromOpenAPISpec()` method so we can delete it. Is that correct?

Reply:
Thanks for the quick response, I chose to rename the method as removeTokensFromSpec that way it remains agnostic to the type of specification, whether openapi or AsyncAPI. On the other hand, this method effectively replaces removeExamplesFromOpenAPISpec() so I follow your advice to eliminate it